### DISCUSSION TITLE
Community RFC Call - March 22, 2021

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#152] [Announcements]
# To Discuss

- https://github.com/withastro/rfcs/pull/150
- https://github.com/withastro/rfcs/pull/147
- https://github.com/withastro/rfcs/pull/153

_Is something that you would like to discuss? Comment below! We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)._

**Comment by matthewp** [comment]
I'd like to discuss this RFC to change `Astro.request`: https://github.com/withastro/rfcs/pull/153

**Reply by FredKSchott** [reply]
Added!

**Comment by natemoo-re** [comment]
The #150 RFC has been updated to address feedback from last week. https://github.com/withastro/rfcs/pull/150/commits/8bd399a41d17d2f845159af1b65c7cf6ce045da0
